From e33517b006f3570d720c209cb24d53f4b78f18fe Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Thu, 25 Feb 2016 15:49:50 +0100
Subject: [PATCH 0099/1345] ARM: mvebu: add rtc dts node for the Armada 395
 SoC family

This commit refers to SDK commit 1f68b82cdf57("ARM: mvebu: add support
for the Armada 395 SoC family") from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/armada-395.dtsi |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/armada-395.dtsi b/arch/arm/boot/dts/armada-395.dtsi
index bf7e433..d63d8aa 100644
--- a/arch/arm/boot/dts/armada-395.dtsi
+++ b/arch/arm/boot/dts/armada-395.dtsi
@@ -56,6 +56,13 @@
 				reg = <0x18000 0x20>;
 			};
 
+			rtc@a3800 {
+				compatible = "marvell,armada-380-rtc";
+				reg = <0xa3800 0x20>, <0x184a0 0x0c>;
+				reg-names = "rtc", "rtc-soc";
+				interrupts = <GIC_SPI 21 IRQ_TYPE_LEVEL_HIGH>;
+			};
+
 			sata@a8000 {
 				compatible = "marvell,armada-380-ahci";
 				reg = <0xa8000 0x2000>;
-- 
1.7.9.5

